docs/dotnet.rst

```diff
@@ -0,0 +1,134 @@
+Dotnet
+======
+
+Prerequisites
+~~~~~~~~~~~~~
+- The application is built with a Linux-compatible .NET desktop application framework such as:
+    - `Avalonia UI <https://avaloniaui.net/>`_
+    - `Uno Platform <https://platform.uno/>`_
+    - `Eto <https://github.com/picoe/Eto>`_
+    - `GTKSharp <https://github.com/GtkSharp/GtkSharp>`_
+- The application's source code is hosted on a Git server such as GitHub, GitLab, or Bitbucket
+- A Git commit with a `tag <https://git-scm.com/book/en/v2/Git-Basics-Tagging>`_ exists on the repository to use for the build. This usually represents the version number of the application (e.g. v1.0, v1.1, etc.).
```

Comment:
```suggestion
```

You can remove this two, it doesn't have to have a tag and be hosted somewhere. Local sources and commits work too.

Reply:
Can you also add going through the "Building your first Flatpak" as a prereq ? https://docs.flatpak.org/en/latest/first-build.html

Reply:
Perhaps including tagging is a bit superfluous for a first-time tutorial, so I've just removed it!

However, I've kept the remote git repository as a pre-requisite, since it's just a pre-requisite for this guide. I did attempt and succeed at locally packaging using a local git repo, but the additional steps required to set up the directory tree did make things a bit harder to follow.